### ISSUE TITLE
Hitting back from blog article page some times does nothing

### ISSUE THREAD

**Issue by yknl** [blockstack/blockstack.org#608]
Steps to reproduce:
1) Go to /blog
2) Click on a blog article
3) Hit back
4) Repeat from 2) until error

Console error:
```
_app.js:1 Uncaught TypeError: Failed to construct 'URL': Invalid URL
    at _app.js:1
    at _app.js:1
    at _app.js:1
    at Object.doUpdateUrl (_app.js:1)
    at _app.js:1
(anonymous) @ _app.js:1
(anonymous) @ _app.js:1
(anonymous) @ _app.js:1
(anonymous) @ _app.js:1
(anonymous) @ _app.js:1
main-39939ed85ac21612474e.js:1 Uncaught (in promise) TypeError: Cannot read property 'split' of undefined
    at e._rewriteUrlForNextExport (main-39939ed85ac21612474e.js:1)
    at t.<anonymous> (main-39939ed85ac21612474e.js:1)
    at L (main-39939ed85ac21612474e.js:1)
    at Generator._invoke (main-39939ed85ac21612474e.js:1)
    at Generator.t.(/blog/anonymous function) [as next] (https://deploy-preview-569--blockstackorgv2.netlify.com/_next/static/commons/main-39939ed85ac21612474e.js:1:378711)
    at u (main-39939ed85ac21612474e.js:1)
    at s (main-39939ed85ac21612474e.js:1)
    at main-39939ed85ac21612474e.js:1
    at new Promise (<anonymous>)
    at new e (main-39939ed85ac21612474e.js:1)
```

**Comment by wbobeirne** [comment]
As far as I can tell, it looks like this is an issue between `redux-bundler` and `next`. According to [this github issue](https://github.com/HenrikJoreteg/redux-bundler/issues/6), it looks like redux-bundler doesn't like server side rendered applications because its `url` reducer tries to maintain routing history in its state, and listens to popState, but next is doing some handling of its own there.

I don't see us actually using the built in redux-bundler stuff here, so I think I'm just going to remove most of it (by using `composeBundlesRaw` instead of `composeBundles`.) I'll test around on the site to make sure we weren't using it anywhere that I missed.

**Comment by wbobeirne** [comment]
Closed via 3b378d928972b4a935c91f5ea87885814d46e7cc.
